### PR TITLE
hponcfg: Configure HP iLO mgmt board (in-band)

### PR DESCRIPTION
**Pull request by dagwieers**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

hponcfg

##### ANSIBLE VERSION

v2.3

##### SUMMARY

This is the original `hponcfg` module that was once accepted in
Ansible but had been removed subsequently because it could not be tested
by the Ansible project.

Since then it was moved to the ansible-provisioning project and
maintained by HP engineers going forward.

Now we are trying to get it upstreamed again.

### CODE REVIEW

File: lib/ansible/modules/remote_management/hpilo/hponcfg.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        </login>
+      </ribcl>
+    dest: /tmp/enable-ssh.xml
+
+- name: Configure HP iLO using enable-ssh.xml
+  hponcfg:
+    src: /tmp/enable-ssh.xml
+'''
+
+import subprocess
```
Comment (resmo) [inline lib/ansible/modules/remote_management/hpilo/hponcfg.py]: Seems unused, is it?

Comment (dagwieers) [reply lib/ansible/modules/remote_management/hpilo/hponcfg.py]: Indeed, has been unused for 5 years ;-) Thanks !

**Review by resmo — APPROVED** [review]
based on code review LGTM
shipit

**Comment by dagwieers** [comment]
cc: @amcguign

**Review by gundalow — APPROVED** [review]

**Pull request merged**